merlin/examples/workflows/flux/scripts/flux_info.py

```diff
@@ -68,24 +69,28 @@
 except BaseException:
     top_dir = "job"
 
-    def get_data_dict(key):
-        kwargs = {
+    def get_data_dict(key) -> Dict:
```

Comment:
i think key is a str

Reply:
change to key:str

Reply:
I changed this the day the comment was made, it shows it correctly in the Files changed tab, but it's not updating here, the comments are remaining tied to the commit they were made with and not updating to the code they're commenting on in the current commit.